README.md

```diff
@@ -8,10 +8,19 @@ audio streams.
 There exists a protocol definition (below), a Go library, and Asterisk
 application and channel interfaces.
 
-**NOTE:**  as of 2020-01-15, AudioSocket has been included in the upstream Asterisk
-system.  While I am leaving the Asterisk patches here for use with previous
-versions, the Asterisk code in this repository should be considered obsolete.
-The Go code is up-to-date, maintained, and this is the primary source for it.
+**NOTE:** as of 2020-01-15, AudioSocket has been included in the upstream Asterisk
+system.  
+
+**NOTE** February 2025:
```

Comment:
I think this can be reduced to just being a note about the DTMF support.  Otherwise, it's just a diff list which, after this merges, will no longer be different, right?

Reply:
Sure! I have removed those and left the note about the DTMF patch for Asterisk.